redisenterprise/assets/configuration/spec.yaml

```diff
@@ -0,0 +1,37 @@
+name: Redis Enterprise
+files:
+- name: redisenterprise.yaml
+  options:
+  - template: init_config
+    options:
+    - template: init_config/http 
+    - template: init_config/default
+  - template: instances
+    options:
+    - name: host
+      required: true
+      description: The RedisEnterprise host
+      value:
+        type: string
+        example: myrediscluster.example.com
+    - name: port
+      required: false
+      description: The RedisEnterprise API port - default 9443
+      value:
+        type: integer
+        example: 9443
+    - name: event_limit
+      required: false
+      description: Number of events to fetch - default 50
+      value:
+        type: integer
+        example: 50
+    - template: instances/http
+      overrides:
+          username.description: The RedisEnterprise API user
+          username.required: true
+          username.value.example: redisadmin@example.com
+          password.required: true
+          password.description: The RedisEnterprise API user password
+          tls_verify.value.default: false
```

Comment:
```suggestion
          tls_verify.value.default: false
          tls_verify.value.example: false
```

Reply:
@coignetp Even if I set the two above to false and run

```
@pytest.fixture
def instance():
    return {
        'host': get_docker_hostname(),
        'port': 9443,
        'username': BOOTSTRAP['credentials']['username'],
        'password': BOOTSTRAP['credentials']['password'],
    }
```

I receive an SSL error

```
tests/test_redisenterprise.py:23: in test_version
    aggregator.assert_service_check('redisenterprise.running', RedisenterpriseCheck.OK)
.tox/py38/lib/python3.8/site-packages/datadog_checks/base/stubs/aggregator.py:324: in assert_service_check
    self._assert(
.tox/py38/lib/python3.8/site-packages/datadog_checks/base/stubs/aggregator.py:333: in _assert
    assert condition, new_msg
E   AssertionError: Needed at least 1 candidates for 'redisenterprise.running', got 0
E     Expected:
E             ServiceCheckStub(check_id=None, name='redisenterprise.running', status=0, tags=None, hostname=None, message=None)
E     Similar submitted:
E     Score   Most similar
E     0.75    ServiceCheckStub(check_id='', name='redisenterprise.running', status=2, tags=[], hostname='', message="HTTPSConnectionPool(host='localhost', port=9443): Max retries exceeded with url: /v1/cluster (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: self signed certificate (_ssl.c:1131)')))")
E   assert False
```

So it would appear that the configuration file changes, but the actual setting to set tls_verify by default is not really working.  This doesn't seem to be picked up at the wrapper level.  Please advise.

Reply:
You're right, it's possible to add a configuration remapper to change the default value in the python code too, see suggestion https://github.com/DataDog/integrations-extras/pull/849#discussion_r665215067